cmd/prReview.go

```diff
@@ -26,6 +29,8 @@ var prReviewCmd = &cobra.Command{
 			return
 		}
 
+		saveOutput, _ := cmd.Flags().GetBool("output")
```

Comment:
It's good practice to update tests whenever the business functionality changes. Ensure that test cases are updated or newly added according to the new functionality of this PR.